docs/docs-beta/src/components/CodeExample.tsx

```diff
@@ -78,18 +80,18 @@ function useLoadModule(
   startAfter: string,
   endBefore: string,
 ) {
-  const isServer = typeof window === 'undefined';
-  if (isServer) {
-    const module = require(`!!raw-loader!/../../examples/${path}`);
-    processModule({cacheKey, module, lineStart, lineEnd, startAfter, endBefore});
-  }
+  //const isServer = typeof window === 'undefined';
+  //if (isServer) {
+  //  const module = CODE_EXAMPLE_PATH_MAPPINGS[path];
+  //  processModule({cacheKey, module, lineStart, lineEnd, startAfter, endBefore});
+  //}
 
   if (!contentCache[cacheKey]) {
```

Comment:
This means that docusaurus supports Suspense on the server!

Reply:
(the fact that we didn't need the isServer logic).

Reply:
heck yeah! 🤯

Reply:
should've probably removed that comment, whoops. Can do that in the future.